packages/scripts/src/generate-readme/generator.ts

```diff
@@ -135,6 +154,8 @@ export class ReadmeGenerator {
       ).settingsDefinition
       this.name = adapter.name
       this.envVars = adapterSettings || {}
+      const rateLimits = adapter.rateLimiting?.tiers || {}
+      this.rateLimits = rateLimits
```

Comment:
This is just me nitpicking but could we do this in a single line?
```suggestion
      this.rateLimits = adapter.rateLimiting?.tiers || {}
```